pkg/notifiers/metadata_getter.go

```diff
@@ -11,4 +11,5 @@ import (
 //go:generate mockgen-wrapper MetadataGetter
 type MetadataGetter interface {
 	GetAnnotationValue(ctx context.Context, alert *storage.Alert, annotationKey, defaultValue string) string
+	GetNamespaceLabels(ctx context.Context, alert *storage.Alert) map[string]string
```

Comment:
GetLabelValue? Or is that not the intent - and the intent is return all labels?

Reply:
Yes the request is to return _all_ of them